### PR TITLE
Second round of style conventions edits for: "How to Deploy a Rollup chain using the Orbit SDK"

### PR DESCRIPTION
**Pull request by anegg0**
Goal of this PR: getting content aligned with OCL's [style guide](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#b420b2b1ff614c71b01743e1b140ffbe)

### CODE REVIEW

**Review by GreatSoshiant — APPROVED** [review]
LGTM!

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ author: GreatSoshiant
 sme: GreatSoshiant
 target_audience: 'Developers deploying and maintaining Orbit chains.'
 sidebar_position: 2
+user_story: As a current or prospective Orbit chain deployer, I need to understand how to deploy an Orbit Rollup chain the Orbit SDK by understanding the available onboarding paths.
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md:9]:
```suggestion
user_story: As a current or prospective Orbit chain deployer, I need to understand how to deploy an Orbit Rollup chain using the Orbit SDK.
```
maybe this is the intended copy?

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md [hunk 2]
```diff
@@ -32,106 +34,106 @@ These contracts are deployed on <a data-quicklook-from="parent-chain">`parent ch
...
-    uint256 delaySeconds;
-    uint256 futureSeconds;
-}
-```
-
-This nested `struct` within `Config` specifies time variations related to block sequencing, providing control over block delay and future block settings.
+### Rollup deployment parameters
+
+[`createRollup`](https://github.com/OffchainLabs/nitro-contracts/blob/acb0ef919cce9f41da531f8dab1b0b31d9860dcb/src/rollup/RollupCreator.sol#L107) is the function that will deploy your core contracts on the parent chain.
+`createRollup` takes a complex input named `deployParams`defining the characteristics of an Orbit Rollup chain
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md:40]:
```suggestion
`createRollup` takes a complex input named `deployParams` defining the characteristics of an Orbit Rollup chain.
```

Comment (anegg0) [reply arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md]: Merged, thank you!

**Pull request merged**